src/httprequest.rs

```diff
@@ -470,7 +472,9 @@ mod tests {
         let (router, _) = Router::new("/prefix/", routes);
         let info = router.default_route_info();
         assert!(info.has_route("/user/test.html"));
+        assert!(info.has_route_prefixed("/user/test.html"));
```

Comment:
But we have `/prefix`, why wouldn't we take it into account?
Shouldn't it return false if prefix is missing from path argument?